### PR TITLE
loggers  updated to match to interface

### PR DESCRIPTION
**Pull request by xmlking**

### CODE REVIEW

**Comment by xmlking** [comment]
#### Note: I need to remove `replace github.com/micro/go-micro/v2 => github.com/xmlking/go-micro/v2 v2.0.0-20200218171511-fe3dcc8b0fc3` after code review , before merge.

File: logger/zerolog/zerolog.go [hunk 1]
```diff
@@ -163,20 +156,37 @@ func (l *zeroLogger) Level() logger.Level {
...
 	if level == logger.FatalLevel {
-		exitFunc(1)
+		l.exitFunc(1)
 	}
 }
 
-func (l *zeroLogger) Logf(level logger.Level, format string, args ...interface{}) {
-	l.nativelogger.WithLevel(loggerToZerologLevel(level)).Msgf(format, args...)
+func (l *zeroLogger) Error(level logger.Level, template string, fmtArgs []interface{}, err error) {
+
```
Comment (printfcoder) [inline logger/zerolog/zerolog.go]: plz rm this blank line

File: logger/zerolog/go.mod [hunk 2]
```diff
@@ -2,7 +2,10 @@ module github.com/micro/go-plugins/logger/zerolog/v2
 
 go 1.13
 
+replace github.com/micro/go-micro/v2 => github.com/xmlking/go-micro/v2 v2.0.0-20200218111535-aeee7b0a67d5
```
Comment (printfcoder) [inline logger/zerolog/go.mod]: plz remove this too

**Comment by printfcoder** [comment]
can it be merged?

**Comment by xmlking** [comment]
Waiting for go-micro merge first

**Pull request merged**